hazelcast/src/main/java/com/hazelcast/logging/LoggingService.java

```diff
@@ -25,11 +25,36 @@
 
     void removeLogListener(@Nonnull LogListener logListener);
```

Comment:
nit: it might be a good idea to document this method and `addLogListener`. The same applies to documentation for the interface itself.

Reply:
yeah, there are many missing pieces in IMDG that need improvement. I did the parts I care about and leave others as they are... I don't know much about this method and never used it, would have to investigate. If I use it one day and find something surprising, i'll document it.

Reply:
In other words, I'm unable to say more than "Removes the given log listener" without checking the code and behavior. And that's zero value.